builtin/providers/aws/resource_aws_cloudwatch_metric_alarm.go

```diff
@@ -95,6 +96,12 @@ func resourceAwsCloudWatchMetricAlarm() *schema.Resource {
 				Optional:      true,
 				ConflictsWith: []string{"statistic"},
 			},
+			"treat_missing_data": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				Computed:     true,
```

Comment:
btw. new `Default` would probably also require state migration - not sure what's the best option here, but generally speaking I'd try to use `Default` over `Compute` (if there's an option) where possible. I think it makes the behaviour more predictable.